autoload/airline/extensions.vim

```diff
@@ -138,7 +138,9 @@ function! airline#extensions#load()
     call airline#extensions#netrw#init(s:ext)
   endif
 
-  call airline#extensions#ycm#init(s:ext)
+  if exists(':YcmDiag')
```

Comment:
Well, an alternative would be to check for `g:loaded` variable. It probably sets this one up, to prevent resourcing. What is it called?